packages/fxa-settings/src/pages/SigninRecoveryCode/en.ftl

```diff
@@ -0,0 +1,18 @@
+## SigninRecoveryCode
```

Comment:
`Recovery code` naming could be misleading to localizers without any other context in this group comment. Just recently the terms `backup authentication code` and `account recovery key` were created to distinguish the two concepts, so more context would be beneficial to avoid misunderstandings.

Reply:
Thanks for your feedback! I've updated the comments, hopefully it's a bit clearer for localizers - please let me know if it can be improved.

Also: I've added one more Ftl msg with this id: `signin-recovery-code-image-description`